website/pages/gamejam_en.md

```diff
@@ -63,7 +70,7 @@ Everybody is welcome, student and non-student alike. No previous experience is n
 
 <h4 class="highlight">Where are your events held?</h4>
 
-**Due to COVID-19 events might be held remotely via the [LiU Game Jam Discord](https://discord.gg/tP2kDvgQKn), see the specific event you are interested in for more information.**
+**Due to COVID-19 events might be held remotely via the [LiU Game Jam's Discord server](https://discord.gg/tP2kDvgQKn), see the specific event you are interested in for more information.**
```

Comment:
```suggestion
**Due to COVID-19 events might be held remotely via [LiU Game Jam's Discord server](https://discord.gg/tP2kDvgQKn), see the specific event you are interested in for more information.**
```

Reply:
Den här är jag inte så säker på faktiskt. Låter lite konstigt att kombinera the med 's